packages/web-components/package.json

```
@@ -60,6 +60,7 @@
     "doctoc": "doctoc --title '## Table of contents' docs && doctoc --title '## Table of contents' README.md",
     "postinstall": "carbon-telemetry collect --install",
     "start": "cross-env NODE_OPTIONS=--openssl-legacy-provider yarn storybook",
+    "start-dev": "./update-scss-paths.sh & cross-env NODE_OPTIONS=--openssl-legacy-provider yarn storybook",
```

```suggestion
    "start-dev": "./update-scss-paths.sh & yarn start",
```

is it possible to reference the existing `start` script?

---

True, will change to that!